lab_3_generate_by_ngrams/main.py

```diff
@@ -346,7 +509,14 @@ def prune_sequence_candidates(
 
         In case of corrupt input arguments return None.
         """
-
+        if not (isinstance(sequence_candidates, dict)
+                and sequence_candidates):
+            return None
+        sorted_sequence_candidates = dict(sorted(list(sequence_candidates.items()),
```

Comment:
Dictionaries do not guarantee order.

Reply:
I tried to fix it but I  think that I'm wrong.  I tried to do sorting without sorted but it didn' work